Makefile

```
@@ -233,14 +233,24 @@ install-resources:
 e2e-dependencies:
 
 K8SCLIENT ?= oc
+GINKGO = $(LOCAL_BIN)/ginkgo
+IS_HOSTED ?= false
+MANAGED_CLUSTER_NAMESPACE ?= managed
```

Do you need to set a default of `CLUSTER_NAMESPACE_ON_HUB` such as?
```suggestion
MANAGED_CLUSTER_NAMESPACE ?= managed
CLUSTER_NAMESPACE_ON_HUB ?= $(MANAGED_CLUSTER_NAMESPACE)
```

---

common.go I set ```
func InitInterfaces(hubConfig, managedConfig string, isHosted bool) {
	if isHosted {
		ClientHosting = NewKubeClient("", hubConfig, "")
		ClientHostingDynamic = NewKubeClientDynamic("", hubConfig, "")
	} else {
		ClientHosting = NewKubeClient("", managedConfig, "")
		ClientHostingDynamic = NewKubeClientDynamic("", managedConfig, "")
	}

	if ClusterNamespaceOnHub == "" {
		ClusterNamespaceOnHub = ClusterNamespace
	}

	...
```
actually you set this on integration_suite_test.go e2e_suite_test.go but I moved to there because it is duplicated